packages/rancher-sachet/charts/values.yaml

```diff
@@ -0,0 +1,84 @@
+# Default values for sachet.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+  namespaceOverride: ""
+
+nameOverride: "sachet"
+fullnameOverride: ""
+
+sachet:
+  # reference: https://github.com/messagebird/sachet/blob/master/examples/config.yaml
+  providers: {}
+
+  receivers: []
+
+replicaCount: 1
+
+image:
+  repository: rancher/mirrored-messagebird-sachet
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 0.2.3
+
+imagePullSecrets: []
+
+podAnnotations: {}
+
+podSecurityContext:
+# fsGroup is the group ID associated with the container
+# fsGroup: 2000
+# Security Context properties
+
+securityContext:
+  # runAsUser is the user ID used to run the container
+  runAsUser: 1000
+  runAsNonRoot: true
+  # runAsGroup is the primary group ID used to run all processes within any container of the pod
+  runAsGroup: 1000
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+
+service:
+  type: ClusterIP
+  port: 9876
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
```

Comment:
whats this ?

Reply:
those are the comments from Helm's default template. 
I will remove them.